packages/app/src/components/inspector/FillRow.tsx

```diff
@@ -79,7 +80,10 @@ export default memo(function FillRow({
         />
         <Spacer.Horizontal size={8} />
         <InputField.Root id={hexInputId} labelPosition="start">
-          <InputField.Input value={'FFFFFF'} onSubmit={() => {}} />
+          <InputField.Input
+            value={`${sketchColorToHex(color).replace('#', '').toUpperCase()}`}
```

Comment:
We can probably move `toUpperCase()` directly into the `sketchColorToHex` function. I think we always want uppercase in this app